src/vtrasterizer/RenderTarget.h

```diff
@@ -9,23 +9,24 @@
 #include <vtrasterizer/TextureAtlas.h>
 #include <vtrasterizer/shared_defines.h>
 
+#include <crispy/ImageSize.h>
 #include <crispy/size.h>
 
 #include <array>
 #include <memory>
 #include <optional>
 #include <vector>
 
-#include "crispy/ImageSize.h"
-
-namespace terminal
+namespace vtbackend
 {
 struct BackgroundImage;
 }
 
-namespace terminal::rasterizer
+namespace vtrasterizer
 {
 
+using ImageSize = crispy::image_size;
```

Comment:
You changed it to direct type `crispy::image_size` inside `Pixmap.h`

Reply:
i know. i just didn't decide yet :)

Reply:
i now decided to move ImageSize/Width/Height into vtpty, because - just like PageSize - it's needed to configure as well as expose the pixel dimensions of the terminal.

only one problem remains: `text_shaper` doesn't make sense to get that from vtpty, for obvious reasons. I think I'll simply dup it here.